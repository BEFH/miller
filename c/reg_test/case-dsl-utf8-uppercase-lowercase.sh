run_mlr --icsvlite --opprint put '$langue = toupper($langue)' $indir/utf8-1.csv
run_mlr --icsvlite --opprint put '$nom    = toupper($nom)'    $indir/utf8-1.csv
run_mlr --icsvlite --opprint put '$jour   = toupper($jour)'   $indir/utf8-1.csv

run_mlr --icsvlite --opprint put '$langue = capitalize($langue)' $indir/utf8-1.csv
run_mlr --icsvlite --opprint put '$nom    = capitalize($nom)'    $indir/utf8-1.csv
run_mlr --icsvlite --opprint put '$jour   = capitalize($jour)'   $indir/utf8-1.csv
