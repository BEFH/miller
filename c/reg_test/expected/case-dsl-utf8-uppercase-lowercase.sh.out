
mlr --icsvlite --opprint put $langue = toupper($langue) ./reg_test/input/utf8-1.csv
langue   nom      jour
FRANÇAIS françois vendredi
ŽUSZ     éloignée -

mlr --icsvlite --opprint put $nom    = toupper($nom) ./reg_test/input/utf8-1.csv
langue   nom      jour
français FRANÇOIS vendredi
žusz     ÉLOIGNÉE -

mlr --icsvlite --opprint put $jour   = toupper($jour) ./reg_test/input/utf8-1.csv
langue   nom      jour
français françois VENDREDI
žusz     éloignée -

mlr --icsvlite --opprint put $langue = capitalize($langue) ./reg_test/input/utf8-1.csv
langue   nom      jour
Français françois vendredi
Žusz     éloignée -

mlr --icsvlite --opprint put $nom    = capitalize($nom) ./reg_test/input/utf8-1.csv
langue   nom      jour
français François vendredi
žusz     Éloignée -

mlr --icsvlite --opprint put $jour   = capitalize($jour) ./reg_test/input/utf8-1.csv
langue   nom      jour
français françois Vendredi
žusz     éloignée -

